src/App.js

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
+
 import TodoList from "./TodoList";
 import AddTodoform from "./AddTodoForm";
 function App() {
-  const [newTodo, setNewTodo] = React.useState("")
+  const[todoList, setTodoList] = React.useState([]);
```

Comment:
same as above, can import useState from React library so you don't need to reference it.